### PR TITLE
Updated Vagrantfile and Ansible playbooks for general use

### PR DESCRIPTION
**Pull request by Ceiu**
- Updated the Candlepin Vagrantfile and candlepin.yml playbooks
  to be more compatible with manual invocation on arbitrary
  systems
- Renamed the "vagrant" directory to "ansible" to better align
  with its contents

### CODE REVIEW

File: ansible/candlepin.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    #
+    # [1] https://docs.ansible.com/ansible/latest/user_guide/playbooks_reuse_roles.html
+    - block:
+        - name: "Include custom role if present"
+          include_role:
+            name: "{{cp_custom_role}}"
+          vars:
+            cp_custom_role: candlepin_dev
+      rescue:
+        - debug:
```
[the remark this answers is not in the document]
Comment (Ceiu) [reply ansible/candlepin.yml:22]: Ugh, this doesn't work as intended. Updating accordingly.

[the remark this answers is not in the document]
Comment (Ceiu) [reply ansible/candlepin.yml]: Fixed... kind of.

**Review by nikosmoum — CHANGES_REQUESTED** [review]
There are some references to the `vagrant/` directory in the README, that need updating to `ansible/`

**Comment by Ceiu** [comment]
@nikosmoum fixed

**Review by nikosmoum — APPROVED** [review]

**Comment by Ceiu** [comment]
The periodic failure of the ConsumerResource test and the SonarQube failures have no barring on the changes here. I will address the former in a separate PR because I'm sick of seeing it, and I believe others are already addressing the SonarQube issues.

**Pull request merged**